src/main/java/org/repodriller/util/SimpleCommandExecutor.java

```diff
@@ -7,24 +7,68 @@
 import java.util.List;
 import java.util.Scanner;
 
+/**
+ * If your analysis needs to work with external tools, a SimpleCommandExecutor gives you a clean way to interact with them.
+ * Usage:
+ *  - Set environment vars with the various manipulation methods.
+ *  - Execute with {@link SimpleCommandExecutor#execute}.
+ *
+ * @author Mauricio Aniche
+ */
 public class SimpleCommandExecutor {
 
-	private List<EnvironmentVar> vars = null;
-	
-	public void setEnvironmentVar(String name, String value)
-	{
-		if( vars == null )
-			vars = new ArrayList<EnvironmentVar>();
-		
-		vars.add(new EnvironmentVar(name, value));
+	private List<EnvironmentVar> envVars = null;
+	private boolean inheritEnv = false;
+
+	public SimpleCommandExecutor() {
+		envVars = new ArrayList<EnvironmentVar>();
+		inheritEnv = false;
+	}
+
+	/**
+	 * Should child inherit parent's environment vars?
+	 *
+	 * @param inherit	True for inherit, else false.
+	 * @return 			this, for chaining
+	 */
+	public SimpleCommandExecutor inheritEnv(boolean inherit) {
```

Comment:
do we have a need for this feature?

Reply:
The current implementation does this implicitly when no environment variables are specified (it returns null when the list is empty). I thought it would be nice to make that behavior explicit.

Reply:
Good idea, thanks!